pyproject.toml

```diff
@@ -18,7 +18,7 @@ flake8-import-order = "^0.18.1"
 flake8-print = "^3.1.4"
 pytest = "^5.4.3"
 pytest-cov = "^2.10.0"
-# PyQt5 = "^5.15.4"  # needed for PyQt example
+PySide6 = "^6.0.2"
```

Comment:
@Korijn Do you want to keep this commented out? I guess this was commented out to save some time on CI?

Reply:
Nope, this is fine! Not sure why it was commented out